core/domain/change_domain.py

```diff
@@ -125,7 +125,10 @@ class BaseChange:
         'user_id_attribute_names': []
     }]
 
-    def __init__(self, change_dict: Dict[str, str]) -> None:
+    def __init__(
+        self,
+        change_dict: Dict[str, Any]
```

Comment:
Why is this `Any` ? 
What are the possible types of values for this dict?

Reply:
Since we know all the types of values, can't make a new type and use it rather than `Any` then?